components/eamxx/docs/user/ml_correction.md

```diff
@@ -0,0 +1,35 @@
+# ML Correction in EAMxx
+
+Machine learning correction is used to emulate lower resolution simulation to behave like its higher resolution counterpart. This project is under development and is only supported on selected machines.
+
+## Example setup
+
+To enable ML correction as a process:
+
+```shell
+./atmchange physics::atm_procs_list="mac_aero_mic,rrtmgp,mlcorrection"
+```
+
+The following options can be specified:
+
+```shell
+./atmchange ML_model_path_tq=/path/to/pretrained/temperature_and_specific_humidity_model
+./atmchange ML_model_path_uv=/path/to/pretrained/u_and_v_model
```

Comment:
Do we still have the Tuv only option, or did we drop that since it wasn't showing positive results?

Reply:
We do, including in the doc now.